__tests__/ledger/LedgerArk.test.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const LedgerArk = require('../../lib/ledger/LedgerArk.js')
+
+describe('LedgerArk', () => {
+  it('should be defined', () => {
+    expect(LedgerArk).toBeDefined()
+  })
+})
+
+describe('LedgerArk.setComm', () => {
+  it('should be defined', () => {
+    expect(LedgerArk.setComm).toBeDefined()
```

Comment:
Rather then simply expecting it to be defined we should expect it to be a function.

https://github.com/jest-community/jest-extended#tobefunction